src/mozanalysis/experiment.py

```diff
@@ -593,43 +602,62 @@ def _build_analysis_windows_query(analysis_windows) -> str:
     def _build_enrollments_query(
         self,
         time_limits: TimeLimits,
-        enrollments_query_type: str,
+        enrollments_query_type: EnrollmentsQueryType,
         sample_size: int = 100,
     ) -> str:
         """Return SQL to query a list of enrollments and their branches"""
-        if enrollments_query_type == "normandy":
+        if enrollments_query_type == EnrollmentsQueryType.NORMANDY:
             return self._build_enrollments_query_normandy(time_limits, sample_size)
-        elif enrollments_query_type == "glean-event":
+        elif enrollments_query_type == EnrollmentsQueryType.GLEAN_EVENTS:
             if not self.app_id:
                 raise ValueError(
                     "App ID must be defined for building Glean enrollments query"
                 )
             return self._build_enrollments_query_glean_event(
                 time_limits, self.app_id, sample_size
             )
-        elif enrollments_query_type == "fenix-fallback":
+        elif enrollments_query_type == EnrollmentsQueryType.FENIX_FALLBACK:
             return self._build_enrollments_query_fenix_baseline(
                 time_limits, sample_size
             )
+        elif enrollments_query_type == EnrollmentsQueryType.CIRRUS:
+            if not self.app_id:
+                raise ValueError(
+                    "App ID must be defined for building Cirrus enrollments query"
+                )
+            return self._build_enrollments_query_cirrus(
+                time_limits, self.app_id
+            )
         else:
             raise ValueError
 
     def _build_exposure_query(
-        self, time_limits: TimeLimits, exposure_query_type: str
+        self, time_limits: TimeLimits, exposure_query_type: EnrollmentsQueryType
     ) -> str:
         """Return SQL to query a list of exposures and their branches"""
-        if exposure_query_type == "normandy":
+        if exposure_query_type == EnrollmentsQueryType.NORMANDY:
             return self._build_exposure_query_normandy(time_limits)
-        elif exposure_query_type == "glean-event":
+        elif exposure_query_type == EnrollmentsQueryType.GLEAN_EVENTS:
```

Comment:
I guess this should be `GLEAN_EVENT` and `glean-event` as the string value

Reply:
Oh good catch, I'll update that.